server/repository/src/migrations/v2_04_00/add_cold_storage_type_table.rs

```diff
@@ -14,8 +14,8 @@ impl MigrationFragment for Migrate {
                 CREATE TABLE cold_storage_type (
                     id TEXT NOT NULL PRIMARY KEY,
                     name TEXT NOT NULL,
-                    min_temperature REAL,
-                    max_temperature REAL
+                    min_temperature {DOUBLE},
+                    max_temperature {DOUBLE}
```

Comment:
Gave a runtime error when type was `REAL`, so changed to 'double precision' in keeping with existing values of similar type.